src/data/index.ts

```diff
@@ -114,7 +114,7 @@ export async function generateCollaborationLink() {
 }
 
 async function getImportedKey(key: string, usage: string): Promise<CryptoKey> {
```

Comment:
```suggestion
function getImportedKey(key: string, usage: string) {
```

we can remove the `async` even now, either by changing the return type to `PromiseLike`, or leaving to be inferred as I'm doing above.